lib/models/index.ts

```diff
@@ -63,7 +77,13 @@ export class Models extends Construct {
           DNTK_ALPHA_SCALER: JSON.stringify(0.25),
         }
       });
-
+  
+      if (
```

Comment:
Rather than dumping this if statement in to each model, would it be possible to iterate over the collection stored in the `models` variable at the end? Something like:

```
if (models.length > 0 && props.config.llms?.sagemakerSchedule?.enabled && schedulerRole !== undefined) {
    models.forEach((model) => {
        createStartSchedule(this, id, model.cfnEndpoint, schedulerRole, props.config.llms?.sagemakerSchedule?.sagemakerStartSchedule);
        createStopSchedule(this, id, modelcfnEndpoint, schedulerRole, props.config.llms?.sagemakerSchedule?.sagemakerStopSchedule);
    });
}
```

That way you can do it once at the end and catch all of the models?